apps/addons/fixtures/addons/featured.json

```diff
@@ -193,44 +193,6 @@
         }
     },
     {
-        "pk": 1,
-        "model": "addons.feature",
-        "fields": {
-            "application": 1,
-            "end": "2037-12-01 00:00:00",
-            "created": "2007-10-10 15:34:20",
-            "locale": null,
-            "modified": "2009-12-08 11:00:58",
-            "start": "2007-10-10 15:34:20",
-            "addon": 1003
-        }
-    },
-    {
-        "pk": 2,
-        "model": "addons.feature",
-        "fields": {
-            "application": 1,
-            "end": "2037-12-02 00:00:00",
-            "created": "2007-10-09 15:34:20",
-            "locale": null,
-            "modified": "2009-12-07 11:00:58",
-            "start": "2007-10-09 15:34:20",
-            "addon": 1001
-        }
-    },
-    {
```

Comment:
I'm not sure but I believe this wasn't meant to be deleted (and my guess would be that it would fix all the tests ;)